src/internal/components/pluggableVisualizations/geoChart/PluggableGeoPushpinChart.tsx

```diff
@@ -215,4 +265,24 @@ export class PluggableGeoPushpinChart extends PluggableBaseChart {
 
         return getBucketItemsByType(buckets, BucketNames.COLOR, [METRIC]);
     }
+
+    private updateSuportedProperties(
+        referencePoint: IExtendedReferencePoint,
+        locationItem: any,
```

Comment:
what is the correct type of `locationItem`?

Reply:
`updateSuportedProperties ` -> `updateSupportedProperties ` :)

Reply:
updated